src/server/game/DataStores/DBCStores.cpp

```diff
@@ -20,6 +20,7 @@
 #include "DBCfmt.h"
 #include "Errors.h"
 #include "IteratorPair.h"
+#include "LFGMgr.h"
```

Comment:
LFGMgr on taxi? XD

Reply:
That is correct unless you know of another system Where level checks are done on a player

Reply:
AreaTable has a level (for exploration experience calculation)

Reply:
We got a areastore lookup in the characterhandler portion of this pr. But please do not take that as a arguement. I'll be honest here, the pr works, but just something about just still doesnt feel right to me. I am open maybe some further advice on the pr. Because i dont want to go back adding 400ms sleep thread timers. Haha

Reply:
Well I will absolutely not approve pulling in lfg so there is that argument

Reply:
> Well I will absolutely not approve pulling in lfg so there is that argument

That would be fine and unstandable. Like i said. I was just not certain on the route taken. What would you suggest in this pr?